_posts/2024-04-29-back-from-devoxx.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: blog
+title: Infinispan in Devoxx France 2024
+permalink: /blog/:year/:month/:day/infinispan-devoxx-france
+date: '2024-04-29T00:00:00.000-00:00'
+author: karesti
+tags: [ "conferences", "java", "devoxx", "devoxxfr" ]
+---
+
+== Infinispan 15 in Devoxx France
+
+This month we assisted to one of the biggest developers conference in Europe: Devoxx France in Paris.
+
+Infinispan did not have a specific talk, however it was present indirectly in two talks:
+
+=== Highly available Identity and Access Management with multi-site Keycloak deployments in the cloud
+
+==== Abstract
+A single sign on solution for your customers and employees shouldn't be a single-point-of-failure in
+your architecture. Keycloak, a popular Open Source Identity and Access Management
+solution that provides single sign on, amongst other capabilities, is no exception
+to this.
+A clustered Keycloak deployment in a single site or datacenter provides sufficient
+availability for many. An increasing number of organizations need to utilize multiple
+sites for improved resiliency or to meet legal requirements. In 2023, Keycloak
+overhauled its multi-site capabilities for public cloud infrastructures, tested them
+thoroughly and provided deployment blueprints to the community. They show how to set
+up an AWS infrastructure and deploy Keycloak across multiple sites.
+This talk presents, from an architects and developer perspective, how we approached
+the problem, which architecture we chose, the challenges we faced and which tools
+helped us along the way. Expect to dive into concepts like load shedding, cache stampedes, and automated failover. See tools like Gatling, Helm, OpenTelemetry, Kubernetes Operators and AWS infrastructure in action. We will also provide an outlook for the next steps in our journey.
+These insights will help you to improve your Keycloak deployments as well as design
+and test your own applications so they can withstand high load and site failures.
+
+[caption="Keycloak",link=/assets/images/blog/2024-devoxx-france/keycloak2.jpg]
+image::/assets/images/blog/2024-devoxx-france/keycloak2.jpg[Keycloak]
+
+==== Infinispan
+During this presentation, Alexander Schwartz and Ryan Emerson delve into the latest
+version of Keycloak and the High Availability architecture that facilitates deploying
+two sites with Keycloak.
+
+[caption="Keycloak",link=/assets/images/blog/2024-devoxx-france/keycloak1.jpg]
+image::/assets/images/blog/2024-devoxx-france/keycloak1.jpg[Keycloak]
+
+One key component enabling this architecture is the
+Infinispan Server, with its Cross-Site replication capabilities.
+
+Initially, this capability is realized through Active-Passive replication,
```

Comment:
"this capability is used to provide Active-Passive replication, however the roadmap includes plans for Active-Active replication in the near future".